src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -139,12 +164,14 @@
                     if (data.isFinished) {
                         window.location.href = data.redirectUri;
                     } else {
+                        if (cancelled) return;
```

Comment:
I think we have used multi line if on other plac es, so:

```js
if (cancelled) {
   return;
};
```

Reply:
Yup, i'll try `else if(!loginIsCancelledByUser)` even. Then we wont need the return.